code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -511,4 +511,32 @@
 	ammo_type = /obj/item/ammo_casing/cap
 	matter = list(DEFAULT_WALL_MATERIAL = 600)
 	max_ammo = 7
-	multiple_sprites = 1
\ No newline at end of file
+	multiple_sprites = 1
+
+
+/obj/item/ammo_magazine/ma12gclip
```

Comment:
OH also this should be pathed under /clip as well, because it's a clip of ammo, and not a magazine of its own.

So, /obj/item/ammo_magazine/clip/c12g